internal/namespaces/applesilicon/v1alpha1/custom_server.go

```diff
@@ -36,6 +36,17 @@ var serverStatusMarshalSpecs = human.EnumMarshalSpecs{
 
 func serverCreateBuilder(c *core.Command) *core.Command {
 	c.ArgSpecs.GetByName("type").AutoCompleteFunc = autocompleteServerType
+
+	c.AddInterceptors(func(ctx context.Context, argsI interface{}, runner core.CommandRunner) (i interface{}, err error) {
+		args := argsI.(*applesilicon.CreateServerRequest)
+		bareMetalLink := "\u001B[38;2;121;45;212;4m\u001B]8;;https://console.scaleway.com/organization/contracts\u001B\\Bare Metal Specific Conditions\u001B]8;;\u001B\\\u001B[0m (\u001B[38;2;121;45;212mhttps://console.scaleway.com/organization/contracts\u001B[0m)"
```

Comment:
We should ask only when the contract is not signed. Is there an API call that would allow use to check contracts ?